### ISSUE TITLE
ntp.managed fails on non-english systems

### ISSUE THREAD

**Issue by jfoboss** [saltstack/salt#46504]
### Description of Issue
It is impossible to use `ntp.managed` function on non-english versions of Windows Server 2016. I think the problem is in `win_ntp.py`:
```
if 'command completed successfully' not in ret:
            return False
```
A `command completed successfully` string is only returned in EN version of Windows. In other languages this string differs.

### Setup
```
win_ntp:
  ntp.managed:
    - servers:
      - ntp21.vniiftri.ru,0x8
      - pool.ntp.org,0x8
      - time.windows.com,0x8
```

### Steps to Reproduce Issue
1. Run `ntp.managed` on Windows Server 2016 (EN) and get (as expected):
```
ID: win_ntp
  Function: ntp.managed
    Result: True
```
2. Run the same `ntp.managed` on Windows Server 2016 (RU) and get:
```
ID: win_ntp
  Function: ntp.managed
    Result: False
   Comment: Failed to update NTP servers
```

### Versions Report
Salt Master:
```
Salt Version:
           Salt: 2017.7.4
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: unknown
       dateutil: 1.5
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.8
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.5.1
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.5 (default, Aug  4 2017, 00:39:18)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.3.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4
 
System Versions:
           dist: centos 7.4.1708 Core
         locale: UTF-8
        machine: x86_64
        release: 3.10.0-693.11.6.el7.x86_64
         system: Linux
        version: CentOS Linux 7.4.1708 Core
```

Salt Minion: 2017.7.3

**Comment by gtmanfred** [comment]
That is not great.  If you had time to submit a PR for it, we would greatly appreciate it.

otherwise @team-windows can you take a look at this when you get a chance?

Thanks,
Daniel

**Comment by jfoboss** [comment]
@gtmanfred Sure!

https://github.com/saltstack/salt/pull/46539

**Comment by gtmanfred** [comment]
Awesome! Thanks you very much!

Daniel

**Comment by rallytime** [comment]
Closed via #46539
